version/prop.go

```diff
@@ -40,8 +40,6 @@ func OS() string {
 	return runtime.GOOS
 }
 
-var isSandboxedMacOS lazy.SyncValue[bool]
```

Comment:
Ditto my comment about #7254

Reply:
This one was my addition and it's an alias for two other lazy values https://github.com/tailscale/tailscale/blob/8bb71f48917e619d231d2dc14ed65aac5b1c31f0/version/prop.go#L48
Doesn't seem like we'd be gaining much performance from wrapping this one too.